gateway/api/v1/views.py

```diff
@@ -68,3 +68,16 @@ class FilesViewSet(views.FilesViewSet):
     """
 
     permission_classes = [permissions.IsAuthenticated, IsOwner]
+
+
+class RuntimeJobViewSet(views.RuntimeJobViewSet):  # pylint: disable=too-many-ancestors
+    """
+    RuntimeJob view set.
+    """
+
+    queryset = RuntimeJob.objects.all()
+    serializer_class = v1_serializers.RuntimeJobSerializer
+    permission_classes = [permissions.IsAuthenticated, IsOwner]
```

Comment:
`IsOwner` might not work here because it is checking `owner` or `author` field of object. But `RuntimeJob` does not have that.

We can modify `IsOwner` permission class a little bit to act differently based on object type. Something like
```python
# Instaed of this
class IsOwner(permissions.BasePermission):
    """
    Custom permission to only allow owners of an object to edit it.
    """

    def has_object_permission(self, request, view, obj):
        return obj.author == request.user
        
# do something like
class IsOwner(permissions.BasePermission):
    """
    Custom permission to only allow owners of an object to edit it.
    """

    def has_object_permission(self, request, view, obj):
        if isinstance(obj, RuntimeJob):
            return obj.job.author == request.user
        return obj.author == request.user
```